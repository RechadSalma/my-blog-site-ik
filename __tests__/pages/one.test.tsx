import { render, screen } from "@testing-library/react";
import One from "../../pages/one";

describe("One page testing iK", () => {
  it("renders a heading", () => {
    const { container } = render(<One />);

    const heading = screen.getByRole("heading", {
      name: /page one/i,
    });

    expect(heading).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
