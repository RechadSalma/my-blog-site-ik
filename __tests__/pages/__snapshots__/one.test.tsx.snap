// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`One page testing iK renders a heading 1`] = `
<div>
  <div>
    <a
      href="/two"
    >
      to /two route
    </a>
    <h1>
      page one
    </h1>
  </div>
</div>
`;
